tests/unit/AuxDidAuth.spec.ts

```diff
@@ -1084,6 +1287,29 @@ describe("vidDidAuth", () => {
       );
       expect(response).toBeDefined();
     });
+    it("sign when using did:key", async () => {
+      expect.assertions(1);
+      const { hexPrivateKey, did } = await mockedKeyAndDidKey();
+      const state = DidAuthUtil.getState();
+      const requestPayload: DidAuthTypes.DidAuthRequestPayload = {
+        iss: did,
+        scope: DidAuthTypes.DidAuthScope.OPENID_DIDAUTHN,
+        registration: {
+          jwks_uri: `https://dev.vidchain.net/api/v1/identifiers/${did};transform-keys=jwks`,
```

Comment:
better to always set a first option to check for a env variable or if it is not defined, then use this one. Something like 
`const API_BASE_URL = process.env.WALLET_API_URL || "https://api.vidchain.net"`